pkg/catalogv2/system/system.go

```diff
@@ -284,6 +286,17 @@ func (m *Manager) isInstalled(namespace, name, version string, values map[string
 			return false, err
 		}
 
+		if minVersion != "" {
+			min, err := semver.NewVersion(minVersion)
+			if err != nil {
+				return false, err
+			}
+			if min.LessThan(ver) || min.Equal(ver) {
```

Comment:
So how do you upgrade if you want to? Doesn't this mean that even if you upgrade rancher to a newer version these things will never upgrade?

Reply:
oh right, I think this will not upgrade even if you upgrade rancher.. I wonder if we should just disable that and always require user to manually upgrade.

Reply:
@dramich I am thinking that we can bump the version in here https://github.com/rancher/rancher/pull/31066/files#diff-c55d1ee74f32cd4b9eeeceaec981a180f52e761acb158a5b0b0e088c7725c9caR41 so that when upgrading rancher it will force to upgrade because min Version changed.